src/frontends/ir/src/ir_deserializer.cpp

```diff
@@ -658,12 +663,17 @@ void ov::XmlDeserializer::read_meta_data(const std::shared_ptr<ov::Model>& model
     for (const auto& data : meta_section.children()) {
         if (data.empty())
             continue;
+        // Old version may produce nodes like <name value="..."/>, but it may brake xml-naming convention
+        // Now it should look like <info name="..." value="..."/>.
+        // Also we keep an option to read an old XMLs where it doesn't have name attribute
```

Comment:
In case an attribute can be represented correctly in the old approach, is it also backward compatible in case **if we use old OpenVINO version** (before this PR) and pass a **newly generated IR** with these changes? In case an attribute can be represented correctly in both old and new ways. Could you point me to the original code that makes it possible?

Reply:
@gkrivor, could you reply?

Reply:
@slyalin sorry for the delay, sure. This PR brakes **forward** compatibility, but I'm going to add a check to reduce this. Question isn't about an attribute, question (and original issue) caused by non-compliant tag name. This PR moves a tag name into "name"-attribute.

Reply:
> I'm going to add a check to reduce this

We shouldn't break "forward" compatibility without a good reason. In this case I would expect that we make this change in tag/attribute only for names that really violates xml rule, but for others we need to keep original behavior.